frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -135,17 +143,23 @@ export default function PrimaryDocs({ loaderData, actionData, params }: Route.Co
       </div>
       <Progress className="mt-8" label="" value={30} />
       <PageTitle subTitle={t('protected:in-person.title')}>{t('protected:primary-identity-document.page-title')}</PageTitle>
+
       <FetcherErrorSummary fetcherKey={fetcherKey}>
         <fetcher.Form method="post" noValidate>
-          <InputSelect
-            id="currentStatusInCanada"
-            name="currentStatusInCanada"
-            errorMessage={errors?.currentStatusInCanada?.at(0)}
-            defaultValue={loaderData.defaultFormValues.currentStatusInCanada}
-            required
-            options={currentStatusInCanadaOptions}
-            label={t('protected:primary-identity-document.current-status-in-canada.title')}
-          />
+          <div className="space-y-6">
+            <CurrentStatusInCanada
+              defaultValue={loaderData.defaultFormValues?.currentStatusInCanada}
+              errorMessage={errors?.currentStatusInCanada?.[0]}
```

Comment:
Use [at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at).

```suggestion
              errorMessage={errors?.currentStatusInCanada?.at(0)}
```

Reply:
without at() works... errorMessage={errors?.currentStatusInCanada?.at(0)} cause syntax error